[RHACS][Docs][4.6 and later] ROX-26389: Updating system health documentation (#85632)
openshift/openshift-docs: modules/sensor-upgrade-section.adoc

// Module included in the following assemblies:
//
// * operating/use-system-health-dashboard.adoc

:_mod-docs-content-type: CONCEPT
[id="sensor-upgrade-section_{context}"]
= Sensor upgrade section

The *Sensor upgrade* section organizes the information into the following groups:

> * *Up to date*: Indicates that Sensor is running the latest supported version.

Reviewer: Per style guide (if the GUI has no hyphens, defer to the GUI use but consider mentioning this to engineering)
```suggestion
* *Up-to-date*: Indicates that Sensor is running the latest supported version.
```

> * *Failed*: Indicates that Sensor has encountered issues during the upgrade process, and require immediate attention.

Reviewer: ```suggestion
* *Failed*: Indicates that Sensor has encountered issues during the upgrade process and requires immediate attention.
```

> * *Out of date*: Indicates that Sensor is running on an older version that you need to update.

Reviewer: This is similar to up-to-date above

> * *Unavailable*: Shows components or services that are not reachable or non-functional.

Reviewer: ```suggestion
* *Unavailable*: Shows components or services that are unreachable or nonfunctional.
```
